pkg/logger/logger.go

```diff
@@ -0,0 +1,73 @@
+package logger
+
+import (
+	"context"
+	"encoding/json"
+	"log/syslog"
+
+	"github.com/chainguard-dev/registry-redirect/pkg/syslogger"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"knative.dev/pkg/logging"
+)
+
+type MyLoggingConfig struct {
```

Comment:
Since this is in the `logger` package, `Config` seems sufficient. `logger.Config` is short & expressive.